src/components/FormAlertWrapper.js

```diff
@@ -53,7 +53,7 @@ const defaultProps = {
 // This component takes other components as a child prop. It will then render any wrapped components as a function using "render props",
 // and passes it a (bool) isOffline parameter. Child components can then use the isOffline variable to determine offline behavior.
 const FormAlertWrapper = props => (
-    <View style={[styles.mh5, styles.mb5, styles.flex1, styles.justifyContentEnd, ...props.containerStyles]}>
+    <View style={[props.containerStyles]}>
```

Comment:
shouldn't this be
```
 <View style={props.containerStyles}>
```
or 
```
 <View style={[...props.containerStyles]}>
```

Reply:
Yes. Good point. 

It still works, but I'll make that small change in a follow up PR. Thanks man